File: downtime/GetMySQL.vb
Imports CodeBase2.Databases


Public Class GetMySQL
    Inherits baseMySQL2

    Public Overrides ReadOnly Property GetConnection As MySql.Data.MySqlClient.MySqlConnection
        Get
            Return New MySqlConnection("server=lis-s22104-db1;uid=dniggli;pwd=vvo084;")

        End Get
    End Property


End Class

File: HL7/GetMySQL.cs
﻿using CodeBase2.Databases;
using MySql.Data.MySqlClient;

namespace HL7
{

    public class GetMySQL : baseMySQL2
    {

        public override MySql.Data.MySqlClient.MySqlConnection GetConnection
        {
            get
            {
                return new MySqlConnection("server=lis-s22104-db1;uid=dniggli;pwd=vvo084;");
            }
        }
    }
}